replaced api url

FerdousFardin opened the pull request:
- added 4 new variables called authRoute, orderRoute, historyRoute & menuRoute to their respective paths in the ApiServiceUtils.js file.
- added new URL in the .env file
- replaced all old URLs to new URLs in the ApiServiceUtils.js file
- minor modification of addCategory function in ApiServiceUtils.js file due to not returning proper boolean

srninnovations commented:
Just a note, I have main into our branch, I made a change on main to ignore .env files. 

We shouldn't commit environment files to our repo as it contains sensitive information so we should be careful. Without the .env file if someone gets a copy of our repo, they can't fully run it because they won't have the URL or any secrets. Even though our repo is private, it can lead to accidental exposure. I should have noticed it before really.

srninnovations reviewed (commented):
I have tried to use the app like normal, all seems file. It would be good if you can try out all the options and actions before merging, as a final test.

FerdousFardin commented:
I didn't replace the old URL from the Login.js file which is why the app had errors while logging in (sorry!😥). I also tested the actions thoroughly and everything is working as before so I'm leaving this comment and details what I've done since I'm about to close this PR,
- added a new variable called 'authRoute' and replaced 'apiUrl'  with 'authRoute' at the loginValidate function in the Login.js file
